code/parse/sexp.cpp

```diff
@@ -30858,7 +30865,9 @@ SCP_vector<sexp_help_struct> Sexp_help = {
 		"\t1:\tName of the event in the mission."},
 
 	{ OP_IS_DESTROYED_DELAY, "Is destroyed delay (Boolean operator)\r\n"
-		"\tBecomes true <delay> seconds after all specified ships have been destroyed.\r\n\r\n"
+		"\tBecomes true <delay> seconds after all specified ships have been destroyed.\r\n"
+		"\tWARNING: If this is used multiple times in a directive SEXP unexpected results may be observed. "
+		"Specify multiple parameters instead.\r\n"
```

Comment:
This is a little confusing at first readthrough.  I suggest "WARNING: If multiple is-destroyed-delay SEXPs are used in a directive event, unexpected results may be observed.  Instead, use a single is-destroyed-delay SEXP with multiple parameters."

Reply:
Thanks, I changed it to your text.